pom.xml

```diff
@@ -48,6 +48,19 @@
                 <checksumPolicy>warn</checksumPolicy>
             </releases>
         </repository>
+        <repository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2</url>
```

Comment:
Do we need this repository? I thought maven central is added as default but I might be wrong.

Reply:
For some reason, the dependency could not be resolved initially without adding this. I re-tested and now it works. Removed it